Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM centos:centos7
+
+EXPOSE 8080
+
+ADD sendEmail-epel-7.repo /etc/yum.repos.d/
+
+RUN yum install -y epel-release && \
+    yum -y --setopt=tsflags=nodocs update && \
```

Comment:
Is this necessary? I think it's normally the role of the base image to have up-to-date packages.

I went to look at https://github.com/sclorg/postgresql-container/blob/master/9.5/Dockerfile, and see that we don't run `yum update`.

We do run other things that would be useful here, namely that `rpm -V` to make sure that all packages we passed to `yum install` actually got installed.